.github/workflows/VersionVigilante_bors.yml

```
@@ -5,6 +5,11 @@ on:
     branches:
       - staging
       - trying
+    paths-ignore:
+      - '*.md'
+      - 'bors.toml'
+      - 'codecov.toml'
+      - '.github/*'
```

```suggestion
      - '.github/**'
```
since I want to match files in the workflows subdir too (ref https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags).